util/validate2baseline

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env ruby
```

Comment:
Should this be in a sub-directory of `util` to keep things organized (perhaps `results`)?

Reply:
I am fine with nesting it in a `results` directory.

IMHO, this type of organization doesn't add any value.  When I want to run a command I don't want to type out a multilayered path to said command.  If the `util` directory is where the executables live, then that is as much as I want to type.  This is out of sheer laziness on my part.

```
bundle exec util/validate2baseline
```
vs
```
bundle exec util/results/validate2baseline
```

Having said that, if it makes more sense to @billclaytor and @RandellP than I have no problem with it.

Reply:
I'm thinking of it more from an organizational perspective than an execution perspective (which admittedly may be the wrong perspective). It seems that if all of the scripts were in the `util` directory it would be very cluttered and potentially difficult to find the script you're interested in, especially if a directory listing scrolls offscreen in tmux and I have to do a `control-B / [` and scroll up to find the script I want. Of course this script would always be at or near the bottom of that list ;)

Just trying to keep things standardized and organized.

Reply:
:+1: I have moved the file into `util/results`

Reply:
Thanks!